ex03.ex

```diff
@@ -77,7 +87,15 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  #def list_contains . .. "your code"
```

Comment:
## -2

Here I'd definitely like to see this written without an `if`

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~

I think it is simpler.